README.md

```diff
@@ -398,7 +398,7 @@ In case of an error, the condition `status` will be set to false and `reason` fi
 | ------ | ----------- |
 | Reconciling | The operator is reconciling the desired state with the actual state of the database |
 | ReadSchema | There was an error about reading the schema from ConfigMap or database credentials |
-| GettingDevDB | failed to get the devdb resource, in case we are using the devdb for nomalization
+| GettingDevDB | failed to get the devdb resource, in case we are using the devdb for nomalization |
```

Comment:
```suggestion
| GettingDevDB | Failed to get a [Dev Database](https://atlasgo.io/concepts/dev-database) in case we are using the devdb for nomalization |
```